data/schema/SSVC_Computed_v2.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://democert.org/ssvc/SVC_Computed_v2.schema.json",
+    "title": "Computed SSVC score representing the path in the decision tree",
+    "description": "This schema represents the full path in the decision tree taken by an analyst with a specific role.  The representation of the full decision tree is optional",
+    "type": "object",
+    "properties": {
+	"choices": {
+	    "type": "array",
+	    "items": {
+		"type": "object",
+		"items": {
+		    "type": "string"
+		}
+	    },
+	    "minItems": 1,
+	    "uniqueItems": true		
+	},
+	"computed": {
+	    "description": "Computed score short representation such as SSVCv2/Ps:Nm/T:T/U:E/1605040000/ for a vulnerability with no or minor Public Safety Impact,  total Technical Impact, and efficient Utility, which was evaluated on Nov 10, 2020.",
+	    "type": "string"
+	},
+	"timestamp" : {
+	    "description": "Date and time in ISO format  ISO 8601 format",
+	    "type": "string",
+	    "format": "date-time"
+	},
+	"timestamp_epoch_ms" : {
+	    "description": "Date and time in epocj utc millisec",
```

Comment:
"epocj" should be "epoch"

Reply:
Thanks for this catch.  There is a little more consolidation of information I noticed in the Provision JSON schema.  I will update all this and push a new PR soon out.